File: suma.php
<?php
echo "Ingrese el primer número: ";
$num1 = trim(fgets(STDIN));
echo "Ingrese el segundo número: ";
$num2 = trim(fgets(STDIN));
$suma = $num1 + $num2;
echo "La suma de $num1 y $num2 es $suma\n";
?>

File: suma.c
#include <stdio.h>
int main() {
    int num1, num2, sum;
    printf("Ingrese el primer numero: ");scanf("%d", &num1);
    printf("Ingrese el segundo numero: ");scanf("%d", &num2);
    sum = num1 + num2;
    printf("La suma de %d y %d es: %d\n", num1, num2, sum);
    return 0;
}
